dashboard/app/controllers/application_controller.rb

```diff
@@ -115,7 +115,9 @@ def with_locale
   end
 
   def milestone_response(options)
-    response = {}
+    response = {
+      timestamp: DateTime.now.strftime('%Q').to_i
```

Comment:
This occurs in a couple places, could you add a helper method to compute it?  Looks like it's like Time.now but out to milliseconds instead of seconds, maybe?